### DISCUSSION TITLE
Barcode scanning

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#12851] [Q&A]
From https://github.com/unoplatform/uno/issues/68#issuecomment-1623100372:

@brendanalexdr I am also looking for a solution to read QR codes, but also to read the codes form passports, did you find a solution?

**Comment by jeromelaban** [comment]
At present time, barcode scanning is available for android using https://github.com/Redth/ZXing.Net.Mobile, but it is not available for iOS, macCatalyst (it's not avaible for MAUI either) and other platforms that Uno supports.

Which platforms are you interested in @MarkLFT?

**Reply by MarkLFT** [reply]
Looking for Android and iOS, initially.

**Comment by sb-jeremias** [comment]
I'm from the Scanbot SDK team - we just published a tutorial showing how to add barcode scanning to Uno Platform apps using our .NET MAUI SDK: https://scanbot.io/techblog/uno-platform-barcode-scanner-tutorial/
It works on both Android and iOS from a single codebase. Our SDK also includes MRZ scanning for passports and travel documents if that's what you need.

The guide walks through the complete implementation, and we have a separate tutorial for pure .NET MAUI apps as well.

**Reply by jeromelaban** [reply]
Thanks for sharing your work!

**Reply by sasakrsmanovic** [reply]
Thankls!! Would love to be in touch to bring these news to our joint audiences. Would you please reach out to info@platform.uno and we will coordinate

**Comment by ToddGlodek** [comment]
Possible update : https://github.com/unoplatform/uno/discussions/19686
